### DISCUSSION TITLE
Cube sides hidden in new safari 17.1, potential bug?

### DISCUSSION THREAD

**Discussion by Beauseph** [nolimits4web/swiper#7166] [Q&A]
Hi guys,

I've found that after updating my Safari to 17.1, the sides of the cube effect don't display, or flicker, when it spins
Does anyone else experience the same thing? I'm using the HTML / JS version.

I just want to make sure that i'm not the only one before submitting a bug. 

Thanks in advance!

Best,
Beau

**Comment by Beauseph** [comment]
I've just confirmed that this also happens in the official swiper demo, so i've gone ahead and opened a bug report.

**Comment by HarryYin91610** [comment]
I have found the same bug in our project using swiper cube slide, the 2nd slide will be hidden.
And we found if we change the swiper-wrapper layer's rotateY from 270deg to 270.01deg , this slide will be visible. I'm not sure if it's the safari17's bug.

**Reply by Beauseph** [reply]
Thanks for your reply.

Which version of Swiper did you use when you encountered the problem? And what browser?
Could you pinpoint the exact location where you applied this?

This fix suggests that this is already done, so i dont know if you were using this one at the time. https://github.com/nolimits4web/Swiper/commit/e005b699e7dbd7c343a56f19fca384b28c37cb97
The only problem with this fix is that it only applies this when the browser is safari.

**Reply by Beauseph** [reply]
Hi Harry, still curious to hear about this!
